scripts/release-npm.sh

```diff
@@ -34,11 +37,13 @@ echo "Updating NPM package version to ${TAG}"
 
 mkdir -p dist/
 
+# TODO use the goreleaser-built binaries from the GitHub relase instead of
+# building from scratch
```

Comment:
note

Reply:
Can you open an issue somewhere to track this improvement, if it doesn't already exist?

Reply:
https://snyksec.atlassian.net/browse/CFG-1786